test/service/test_async_manifest_service.py

```diff
@@ -131,7 +131,7 @@ def run(self, result: Optional[unittest.result.TestResult] = ...) -> Optional[un
         with mock.patch('azul.service.manifest_service.ManifestService.__init__') as init_:
             init_.return_value = None
             with mock.patch('azul.service.manifest_service.ManifestService.get_cached_manifest') as get_cache_manifest:
-                get_cache_manifest.return_value = None
+                get_cache_manifest.return_value = None, None  # This returns Tuple[object_key, presign_url] of type str
```

Comment:
I think this comment is incorrect because the return type is a tuple of optional strings. Having said that, I think the comment is unnecessary because the return type for `get_cached_manifest()` is already documented in its definition.

Reply:
Done! :)